apps/frontend/src/composables/useIntlRelativeTimeFormatHumanReadable.ts

```
@@ -0,0 +1,32 @@
+import { computed, toValue, type MaybeRefOrGetter } from 'vue';
+import { useIntlRelativeTimeFormat } from './useIntlRelativeTimeFormat';
+import type { Nil } from '@etonee123x/shared/types';
+import { isNil } from '@etonee123x/shared/utils/isNil';
+
+export const useIntlRelativeTimeFormatHumanReadable = <Since extends number | Nil>(_since: MaybeRefOrGetter<Since>) => {
+  type Return = Since extends number ? string : null;
+
+  const intlRelativeTimeFormat = useIntlRelativeTimeFormat(...([, { numeric: 'auto' }] as const));
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)